internal/contributebot/README.md

```diff
@@ -0,0 +1,5 @@
+# Contribute Bot
+
+Contribute Bot is a small service (written using Go Cloud!) that performs
```

Comment:
How does it use Go Cloud?

Reply:
The event worker (subscriber to the topic) will use Go Cloud. It's a forward-looking statement, happy to remove.